src/routes/api/v1/search-index/+server.ts

```diff
@@ -17,39 +17,23 @@ const CONTENTFUL_ACTIONS = {
   DELETE: "ContentManagement.Entry.delete",
 };
 
-const getMetaTitleAndDescription = (
-  item?: SearchIndexingMetadataMapItem,
-): { metaTitle?: string | null; metaDescription?: string | null } => {
-  let metaTitle, metaDescription;
-  if (item?.__typename == "PageMetadata") {
-    metaTitle = item?.metaTitle ?? item?.title;
-    metaDescription = item?.metaDescription;
-  } else if (item?.__typename === "News") {
-    metaTitle = item?.metaTitle ?? item?.title;
-    metaDescription = item?.metaDescription ?? item?.subhead;
-  } else if (item?.__typename === "EventEntry") {
-    metaTitle = item?.metaTitle ?? item?.shortTitle;
-    metaDescription = item?.metaDescription ?? item?.eventSummary;
-  }
-  return { metaTitle, ...(metaDescription ? { metaDescription } : {}) };
-};
-
 export const POST = async ({ request, locals: { contentfulClient } }) => {
   authenticateRequest(request);
 
   const contentfulAction = request.headers.get("x-contentful-topic") || "";
   const body = await request.json();
   const contentType = body?.sys?.contentType?.sys?.id;
-  const contentTypes = ["pageMetadata", "news", "eventEntry"];
+  // TODO: in the future, we might want to consolidate these search index endpoints
```

Comment:
I basically reverted this endpoint into its original functionality of just handling pageMetadata updates. [I hardened that requirement in the webhook](https://app.contentful.com/spaces/pc5e1rlgfrov/settings/webhooks/0UGOXuEe2TiUSD8dCJjGpS/settings), too, so the original webhook is only getting triggered for PageMetadata Updates